src/components/ha-dialog.ts

```diff
@@ -118,6 +118,22 @@ export class HaDialog extends DialogBase {
         margin-top: var(--dialog-surface-margin-top);
         min-height: var(--mdc-dialog-min-height, auto);
         border-radius: var(--ha-dialog-border-radius, 28px);
+        border-top-left-radius: var(
+          --ha-dialog-border-top-left-radius,
+          var(--ha-dialog-border-radius, 28px)
+        );
+        border-top-right-radius: var(
+          --ha-dialog-border-top-right-radius,
+          var(--ha-dialog-border-radius, 28px)
+        );
+        border-bottom-left-radius: var(
+          --ha-dialog-border-bottom-left-radius,
+          var(--ha-dialog-border-radius, 28px)
+        );
+        border-bottom-right-radius: var(
+          --ha-dialog-border-bottom-right-radius,
+          var(--ha-dialog-border-radius, 28px)
+        );
```

Comment:
Why add all these? We can just do `--ha-dialog-border-radius: 0 0 28px 28px;` right?

Reply:
I didn't wanted to hardcode the `28px` to no break themes using `--ha-dialog-border-radius`.

Reply:
You can also do `--ha-dialog-border-radius: 0 0 var(--ha-dialog-border-radius) var(--ha-dialog-border-radius);` I think.

Reply:
But that would not work if it is not a single value, nevermind

Reply:
Or actually, that is already an issue with your current code 🤔

Reply:
True 😅 I will look to fix that